docs/source/conf.py

```diff
@@ -208,4 +231,5 @@ def setup(app):
     # 2023-06-29 docs refresh
     "topic/cilogon": "tutorials/provider-specific-setup/providers/cilogon",
     "tutorials/provider-specific-setup/providers/awscognito": "tutorials/provider-specific-setup/providers/generic",
+    "how-to/migrations/upgrade-to-16": "reference/changelog",
```

Comment:
```suggestion
```
Is this needed? It's never been released.

Reply:
I guess we can't say for sure, but I guess we can delete it though because the consequences are low at worst.